commit 6d244b7ab3c0dea6855b21bb0f83cb8bb8db4b32
Author: Taylor Hornby <havoc@defuse.ca>
Date:   Wed Sep 9 16:47:41 2015 -0600

    Add an ``LDFLAGS_PREFIX`` for a similar purpose as ``CXXFLAGS_PREFIX``.

diff --git a/Makefile b/Makefile
index 11ba574..daf7d26 100644
--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@ CXXFLAGS += $(CXXFLAGS_PREFIX) -O2 -Wall -Wextra -Wno-unused-parameter -Wno-comm
 DEPSRC=depsrc
 DEPINST=depinst
 
-LDFLAGS += -L $(DEPINST)/lib -Wl,-rpath,$(DEPINST)/lib
+LDFLAGS += $(LDFLAGS_PREFIX) -L $(DEPINST)/lib -Wl,-rpath,$(DEPINST)/lib
 LDLIBS += -lgmpxx -lgmp -lboost_program_options -lcrypto
 CXXFLAGS += -I $(DEPINST)/include -I src -DUSE_ASM -DMONTGOMERY_OUTPUT
 DEFAULT_CURVE=BN128
